addons/website/views/snippets/s_image_text.xml

```diff
@@ -12,7 +12,7 @@
                     <h2>Section Subtitle</h2>
                     <p>Write one or two paragraphs describing your product or services. <br/>To be successful your content needs to be useful to your readers.</p>
                     <p>Start with the customer – find out what they want and give it to them.</p>
-                    <p><a href="#" class="btn btn-primary">Discover more</a></p>
```

Comment:
@Cocographique No, snippets should be able to use btn-primary. What needs to be fixed is the link dialog. I'll leave this out of the PR :+1: